### PR TITLE
React Router and Pages /code

### PR DESCRIPTION
**Pull request by hemant1026**

### CODE REVIEW

File: code/docker-compose.yml
Comment (faizanurfi) [inline code/docker-compose.yml:1]: Revert this change

File: code/packages/react-app/.gitignore
Comment (faizanurfi) [inline code/packages/react-app/.gitignore:1]: Delete ```react-app``` folder from your branch

This folder ```code/packages/react-app``` was renamed to ```code/packages/web```.

So this folder should be deleted altogether

File: code/packages/web/src/App.tsx
Comment (faizanurfi) [inline code/packages/web/src/App.tsx:1]: This file should not have been moved to  ```code/packages/web/src/pages```

Revert this change. ```App.tsx``` should stay in ```code/packages/web/src``` folder

File: code/packages/web/src/pages/App.tsx
Comment (faizanurfi) [inline code/packages/web/src/pages/App.tsx:1]: Move this file to ```code/packages/web/src``` folder. Please see above comment

File: code/packages/web/src/pages/App.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import LandingPage from './components/LandingPage';
+import LoginPage from './components/LoginPage';
+import SignupPage from './components/SignuPage';
+import ForgotPasswordPage from './components/ForgotPasswordPage';
+
+function App() {
+  return (
+    <Router>
```
Comment (faizanurfi) [inline code/packages/web/src/pages/App.tsx]: This will not work unless you wrap `<App />` in `main.tsx` with RouterProvider

Please read React Router documentation at: [React Router](https://reactrouter.com/en/main)

File: code/packages/web/src/pages/ForgotPasswordPage.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const ForgotPasswordPage = () => {
+  return (
+    <div>
+      <h2>Forgot Password?</h2>
+      <form>
```
Comment (faizanurfi) [inline code/packages/web/src/pages/ForgotPasswordPage.tsx]: You should also add a form submit function. Basically form submission should trigger onSubmit which will need to be a function. Leave the function empty for now. Do that for ForgotPasswordPage, LoginPage and SignupPage

**Review by faizanurfi — CHANGES_REQUESTED** [review]

File: code/packages/web/src/pages/ForgotPasswordPage.tsx
Comment (faizanurfi) [inline code/packages/web/src/pages/ForgotPasswordPage.tsx:1]: Create a test file for this as well with a basic test. See `code/packages/src/App.test.tsx` for reference

File: code/packages/web/src/pages/LoginPage.tsx
Comment (faizanurfi) [inline code/packages/web/src/pages/LoginPage.tsx:1]: Create a test file for this as well with a basic test. See `code/packages/src/App.test.tsx` for reference

File: code/packages/web/src/pages/LandingPage.tsx
Comment (faizanurfi) [inline code/packages/web/src/pages/LandingPage.tsx:1]: Create a test file for this as well with a basic test. See `code/packages/src/App.test.tsx` for reference

File: code/packages/web/src/pages/SignupPage.tsx
Comment (faizanurfi) [inline code/packages/web/src/pages/SignupPage.tsx:1]: Create a test file for this as well with a basic test. See `code/packages/src/App.test.tsx` for reference

File: code/docker-compose.yml
Comment (hemant1026) [reply code/docker-compose.yml:1]: This has been updated. Awaiting the pus after the next changes are made

File: code/packages/react-app/.gitignore
Comment (hemant1026) [reply code/packages/react-app/.gitignore:1]: It has been deleted, awaiting the push

File: code/packages/web/src/App.tsx
Comment (hemant1026) [reply code/packages/web/src/App.tsx:1]: It has been moved to code/packages/web/src and removed from code/packages/web/src/pages.

File: code/packages/web/src/pages/App.tsx
Comment (hemant1026) [reply code/packages/web/src/pages/App.tsx:1]: Done

**Review by faizanurfi — APPROVED** [review]

**Review by faizanurfi — APPROVED** [review]

**Review by jaindra-bu — APPROVED** [review]
